src/App.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
```

Comment:
👍 We need a piece of state to track changes to our messages, initialized from the json file we loaded.